src/lib/components/Widget.tsx

```diff
@@ -31,7 +31,7 @@ const WidgetWrapper = styled.div<{ width?: number | string }>`
   font-size: 16px;
   font-smooth: always;
   font-variant: none;
-  height: 356px;
+  height: 360px;
```

Comment:
The extra 4px comes from always showing the TokenButton borders due to the changes in Button. Always showing borders makes handling styles easier, as prior to this borders were special-cased, and required explicit css (eg `calc(1em - 1px)`) to keep box sizes consistent with/without borders.